scripts/infer_patches.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -l
+#SBATCH --job-name=goes16ci
+#SBATCH --account=NAML0001
+#SBATCH --time=12:00:00
+#SBATCH --partition=dav
+#SBATCH --mem=128G
+module purge
```

Comment:
Remove module purge. Do you need a GPU to run this script? If so, `#SBATCH --gres=gpu:v100:1` and `module load cuda/10.1` to the script.